<dec f='include/c++/4.8/bits/functexcept.h' l='91' type='void std::__throw_ios_failure(const char * )'/>
<use f='include/c++/4.8/bits/basic_ios.tcc' l='48' u='c' c='_ZNSt9basic_ios5clearESt12_Ios_Iostate'/>
<use f='include/c++/4.8/bits/fstream.tcc' l='290' u='c' c='_ZNSt13basic_filebuf9underflowEv'/>
<use f='include/c++/4.8/bits/fstream.tcc' l='347' u='c' c='_ZNSt13basic_filebuf9underflowEv'/>
<use f='include/c++/4.8/bits/fstream.tcc' l='351' u='c' c='_ZNSt13basic_filebuf9underflowEv'/>
<use f='include/c++/4.8/bits/fstream.tcc' l='354' u='c' c='_ZNSt13basic_filebuf9underflowEv'/>
<use f='include/c++/4.8/bits/fstream.tcc' l='518' u='c' c='_ZNSt13basic_filebuf22_M_convert_to_externalEPT_l'/>
<use f='include/c++/4.8/bits/fstream.tcc' l='539' u='c' c='_ZNSt13basic_filebuf22_M_convert_to_externalEPT_l'/>
<use f='include/c++/4.8/bits/fstream.tcc' l='601' u='c' c='_ZNSt13basic_filebuf6xsgetnEPT_l'/>
<doc f='include/c++/4.8/bits/functexcept.h' l='89'>// Helpers for exception objects in &lt;ios&gt;</doc>
